\hypertarget{class_json_1_1_exception}{}\section{Json\+:\+:Exception Class Reference}
\label{class_json_1_1_exception}\index{Json\+::\+Exception@{Json\+::\+Exception}}


{\ttfamily \#include $<$json.\+h$>$}



Inheritance diagram for Json\+:\+:Exception\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=288pt]{class_json_1_1_exception__inherit__graph}
\end{center}
\end{figure}


Collaboration diagram for Json\+:\+:Exception\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=167pt]{class_json_1_1_exception__coll__graph}
\end{center}
\end{figure}
\subsection*{Public Member Functions}
\begin{DoxyCompactItemize}
\item 
{\bfseries Exception} (J\+S\+O\+N\+C\+P\+P\+\_\+\+S\+T\+R\+I\+NG const \&msg)\hypertarget{class_json_1_1_exception_ae764aa42e0755bd4ce9d303e2733fa8f}{}\label{class_json_1_1_exception_ae764aa42e0755bd4ce9d303e2733fa8f}

\item 
char const $\ast$ {\bfseries what} () const J\+S\+O\+N\+C\+P\+P\+\_\+\+O\+V\+E\+R\+R\+I\+DE  throw ()\hypertarget{class_json_1_1_exception_a5a9ed7d91b828b9be81706ef9d483ed6}{}\label{class_json_1_1_exception_a5a9ed7d91b828b9be81706ef9d483ed6}

\end{DoxyCompactItemize}
\subsection*{Protected Attributes}
\begin{DoxyCompactItemize}
\item 
J\+S\+O\+N\+C\+P\+P\+\_\+\+S\+T\+R\+I\+NG {\bfseries msg\+\_\+}\hypertarget{class_json_1_1_exception_aae3cbb8b45bf21480f64502a8329659f}{}\label{class_json_1_1_exception_aae3cbb8b45bf21480f64502a8329659f}

\end{DoxyCompactItemize}


\subsection{Detailed Description}
Base class for all exceptions we throw.

We use nothing but these internally. Of course, S\+TL can throw others. 

The documentation for this class was generated from the following files\+:\begin{DoxyCompactItemize}
\item 
/home/david/projects/hdc\+\_\+new/inc/json/json.\+h\item 
/home/david/projects/hdc\+\_\+new/src/jsoncpp.\+cpp\end{DoxyCompactItemize}
